.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -57,23 +60,66 @@ administrator` role:
 
 ## Organization administrator actions
 
-- [ ] Removed the `Azul modify` role (a component of the `operator` role) from
-      the offboarded employee/contractor for all deployments and all zones.
-- [ ] Removed the employee/contractor from the UCSC GitHub organization, or the
-      employee/contractor's employment with the organization has not been
-      terminated.
+- [ ] Removed the offboarded employee/contractor's GitHub account from the
+      `Azul Developers` team in the DataBiosphere organization on GitHub.
+- [ ] Removed the offboarded employee/contractor's Google workspace account from
+      the Google project used for the deployment.
+- [ ] Deleted the Google service account (name derived from the Google workspace
+      account name) in the Google project used for the deployment.
+- [ ] Removed the offboarded employee/contractor's AWS IAM user account in the
+      `gi-gateway` AWS account from the `platform-hca-dev-developer` group (or
+      the `platform-hca-prod-developer` group for a production deployment).
```

Comment:
Wondering if `platform-anvil-dev` should also be added to the list.

Reply:
Hm good catch, I am uncertain. `platform-anvil-dev-developer` is not mentioned in the [source document](https://docs.google.com/document/d/1Kg0dMZmCw6gtkvabD2jYWPZO2Mx_wsC8BJPKdeKTfg0/edit) so I will raise the question to @hannes-ucsc at the next PL.